Khan/perseus
# [Locked Figure Aria] Use spoken math in locked figure settings autogen labels

## Summary:
When auto-generating the aria labels for locked figures, we want it to use
words as if they were spoken rather than math expressions that might be
read incorrectly by the screen reader.

- Use the `generateSpokenMathDetails` utility within LockedVectorSettings,
  LockedEllipseSettings, LockedPolygonSettings, and LockedFunctionSettings.
- Updated LockedFigureAria - remove the prop `prePopulatedAriaLabel` and
  make `getPrepopulatedAriaLabel` required.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2548

## Test plan:
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-vector-settings.test.tsx`
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-ellipse-settings.test.tsx`
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-polygon-settings.test.tsx`
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx`

Storybook
- Go to http://localhost:6006/iframe.html?args=&id=perseuseditor-widgets-interactive-graph--mafs-with-locked-figure-labels-all-flags&viewMode=story
- For each locked figure:
  - Open the figure settings
  - Change the visible label to have a mix of TeX and non-TeX
  - Press the "Auto-generate" button
  - Confirm that the input changes to include spoken math words for the TeX
  - Also try this with no labels, multiple labels, fully non-TeX, and
    fully teX labels.
